### ISSUE TITLE
TypeScript: Missing semicolon

### ISSUE THREAD

**Issue by clebert** [prettier/prettier#1891]
Input: `foo.d.ts`

```ts
declare module 'foo' {
  function foo(namespace: string): void;
}
```

Output:

```ts
declare module 'foo' {
  function foo(namespace: string): void
}
```

Expected:

```ts
declare module 'foo' {
  function foo(namespace: string): void;
}
```

TSLint reports the following error: `Missing semicolon semicolon`

**Comment by vjeux** [comment]
I reproduced this with `semi: true`. cc @azz

**Comment by vjeux** [comment]
Does TSLint complain about

```js
 namespace X {
       export function Y() {
         return 42;
       }
}
```

as well?

**Comment by vjeux** [comment]
What about

```js
export class c1 {}
```

**Comment by vjeux** [comment]
Okay, I think I know what's the issue. It is because it doesn't have a body, then we should add a `;`. But if there are `{}` then no `;`.

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
